### DISCUSSION TITLE
set-cookie header is ignored in response from url

### DISCUSSION THREAD

**Discussion by MirroxDev** [supabase/supabase#22529] [Questions]
I have a problem with OAuth google, if I log into my google account (linked to supabase), then everything works, but if I try to log in from any other account, I don’t receive a session token, and I’m simply redirected to /auth Why does this happen and how to solve it?
Of the errors on the page I noticed in the console: set-cookie header is ignored in response from url. The combined size of the name and value must be then or equal to 4096 characters.

const handleLogin = () => {  
supabase.auth.signInWithOAuth({  
provider: "google",  
options: {  
redirectTo: location.origin + "/auth/callback",  
},  
});  
  };
  
  
"@supabase/ssr": "\^0.0.10",  
"@supabase/supabase-js": "\^2.42.0",

**Comment by GaryAustin1** [comment]
https://github.com/supabase/supabase/issues/18564
https://github.com/supabase/auth-helpers/issues/661
Some relevant info.

**Comment by MirroxDev** [comment]
after 2 days I found a solution to the problem, I don’t quite understand how correct it is but it works.

in files:
```
node_modules\@supabase\ssr\dist\index.js
node_modules\@supabase\ssr\dist\index.mjs
```
I found the line var `MAX_CHUNK_SIZE = 3180;` and replaced it with `MAX_CHUNK_SIZE = 2180;` 


If you know another solution then please write
